opencensus/proto/metrics/BUILD.bazel

```diff
@@ -16,7 +16,7 @@ package(default_visibility = ["//visibility:public"])
 
 proto_library(
     name = "metrics_proto",
-    srcs = ["metrics.proto"],
+    srcs = ["v1/metrics.proto"],
```

Comment:
Better to move the BUILD.bazel in the v1 directory otherwise we should encode the version in the bazel library name.

Reply:
Make sense, done.